gio/src/lib.rs

```diff
@@ -84,19 +84,19 @@ pub use socket_msg_flags::SocketMsgFlags;
 mod subprocess;
 mod subprocess_launcher;
 mod threaded_socket_service;
-#[cfg(any(unix, docsrs))]
+#[cfg(unix)]
```

Comment:
Is this actually correct? This way we won't get docs for this unless building on a unix-y system?

Reply:
I think that is correct yes, but i would refer to @GuillaumeGomez here

Reply:
That seems like a problem then. Previously we got platform-specific docs when building docs on any platform. There's quite a bit of compat hackery for that :)